config/routes.rb

```diff
@@ -1,6 +1,5 @@
 Rails.application.routes.draw do
-  scope "/:locale", locale: /en/ do
-    resources :users, only: [:new]
-    resources :budgets, only: [:new]
-  end
+  resources :users, only: %i[new create show edit update destroy]
```

Comment:
Please change rubocop to change back

Reply:
Done with sha: 5ee4b14c9b19884252e7434c51f40399a08a4cc0